shared/bundle_analysis/parsers/v1.py

```diff
@@ -0,0 +1,371 @@
+import json
+import logging
+import re
+import uuid
+from typing import Tuple
+
+import ijson
+from sqlalchemy.orm import Session as DbSession
+
+from shared.bundle_analysis.models import (
+    Asset,
+    AssetType,
+    Bundle,
+    Chunk,
+    Module,
+    Session,
+    assets_chunks,
+    chunks_modules,
+)
+from shared.bundle_analysis.utils import get_extension
+
+log = logging.getLogger(__name__)
+
+
+"""
+Version 1 Schema
```

Comment:
This schema is what the bundler plugin uploads to storage?

Reply:
yes for version 1, but some plugins can upload version 2 which is a different schema (see v2.py)